config/crd/ec2.aws.crossplane.io_securitygroups.yaml

```diff
@@ -283,7 +283,6 @@ spec:
                         type: array
                     required:
                     - ipProtocol
-                    - ipRanges
```

Comment:
Just a note, I feel like we should formalize our CRD versioning strategy. I know we generally follow the k8s conventions, but we have a few different scenarios that I feel like we encounter. For instance, this is technically a change here but obviously will not break any instances of this CRD that already exist so there is no need for a version bump. Maybe this is obvious, but may be nice for end users to be able to see exactly what we adhere to.

Reply:
I believe we explicitly refer to Kubernetes policy for versioning: https://crossplane.io/docs/master/introduction/managed-resources.html Not sure what additions we would need to make to that though.

Reply:
Yeah basically all I would add is that in a case like this we don't need to bump to `v1beta2` because this does not impact user experience, but maybe that is implied